README.md

```diff
@@ -1,16 +1,16 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/632326528/22.2.4%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T1162029)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
 
 # Use DevExpress MAUI and Blazor Components to Create a .NET MAUI Blazor Hybrid app
 
+
 This demo example shows how to utilize DevExpress Blazor and .NET MAUI components to develop a .NET MAUI Blazor Hybrid app. 
 
 ## Blazor Hybrid View
 
-<img src="media/hybrid-view.png" alt="Views built with the DevExpress for .NET MAUI library" height="400">
+<img src="media/hybrid-view.png" alt=".NET MAUI Blazor Hybrid view" height="400">
 
 This page uses the DevExpress Blazor Grid to display a list of tasks. Adaptive triggers reduce the number of columns and display a column chooser button on smaller screens. The code for this view is shared between MAUI and Blazor Server apps, but MAUI version uses a larger size mode.
```

Comment:
I want to make sure that it's clear that "smaller screens" affect both "number of columns" and "column chooser button", not only the latter. So I move that phrase to the beginning. 

I'm also not sure what you want to say in the last sentence... But all that seems to be a separate thought, so I'm moving it to a separate paragraph. 

```suggestion
This page uses the DevExpress Blazor Grid to display a list of tasks. When the app is viewed on smaller screen sizes, adaptive triggers reduce the number of columns and display a column chooser button. 

The code for this view is shared between MAUI and Blazor Server apps. The MAUI version uses a larger size mode.
```